developing/on-call/README.md

```diff
@@ -0,0 +1,193 @@
+  that you will be taking care of the alert before taking action, so that
+  you are not working at cross-purposes or duplicating work.
+* You should make sure that you are keeping a persistent record of alerts
+  and/or incidents each day. This can be as simple as a Google Doc filled
+  out at the end of the day, but it should record at least the time of the
+  alert, the alert that fired, and what was done to address the alert (even
+  if that is "the alert went away on its own"). This serves as a way to
+  pass knowledge onto the other on-call engineers or the next shift, and
+  allows us to look at the previous week or month for alerts that are
+  particularly troublesome.
+
+## Escalation and Notification Policies
+
+In PagerDuty terms, an escalation policy determines how an alert will
+proceed if it is not acknowledged; a notification policy is something
+which is set for each engineer individually that determines how they will
+be notified if they receive an alert.
+
+* PagerDuty and similar products can be set up to send notifications to
+  Slack for each alert; we recommend doing so. Using the [PagerDuty
+  integration](https://www.pagerduty.com/docs/guides/slack-integration-guide/)
+  will also allow engineers to acknowledge or resolve alerts from Slack
+  if they so choose.
+* Engineers should have notification policies set to ensure that they
+  will be notified within the expect five minute response window. This
+  should use multiple notification methods to make sure things don't fall
+  through the cracks; at least one method should notify you immediately
+  that the alert has fired. Keep in mind that an acknowledgement will break
+  the notification chain. An example might be:
+  * Immediately after the alert, notify me by push notification and email.
+  * 1 minute later, notify via SMS (in case data coverage is bad).
+  * 5 minutes after the alert, notify via voice call.
+* Your on-call rotation should have an escalation policy that escalates
+  from the primary to secondary after no more than 10 minutes, and from
+  secondary to tertiary after no more than an additional 10 minutes.
+  Optimally, this should be as short as possible to ensure that there is
+  a quick response; remember that an alert going unnoticed can incur a
+  significant SLO impact. A 99.99% uptime requires no more than 13
+  minutes of downtime a quarter for instance -- if you have a 10 minute
+  escalation, an alert that falls through to the secondary may blow the
+  SLO on its own if the problem is serious enough.
+
+## Project Expectations
+
+In addition to the expectations we have for on-call engineers, there are
+also expectations we make for the project we are on-call for in order to
+ensure that on-call is not an undue burden.
+
+* The on-call rotation *should not be getting more than 2-3 alerts per
+  day*, and even that is bordering on excessive, especially if these are
+  off-hours. Optimally, this should be no more than 2-3 alerts *per shift*.
```

Comment:
This can be read differently by different people. Slack may get notifications for several things that PagerDuty ends up turning into a single event and paging. Several notifications is different than several pages. I've seen PD and AWS CloudWatch set up in such a way that up to 20 CloudWatch notifications go out (to Slack/Email) but PD only gets one actual page and PD takes care of bundling up the notifications into one event. 

This is all to say that some folks are going to read this and say "notifications count and we should disable so many notifications" when in fact I think we're trying to say "alerts count, the ones that actually get to your phone/sms/voice".

Reply:
Yes, I am talking about alerts here -- I would generally caution against a bunch of unactionable notifications too, at least if they are going through the same channel as alerts, just because it makes alert numbness more likely to set in. Do you think this would benefit from a definition of terms at the beginning?

Reply:
Yeah, common definitions would be super helpful.

Reply:
Added definitions for Alerts and Notifications.